test/pom.xml

```diff
@@ -24,7 +24,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>${junit.version}</version>
+            <scope>compile</scope>
```

Comment:
I don't understand why this change is needed? Isn't `compile` the default scope?